interfaces/Portalicious/src/app/pages/project-registration-activity-log/components/table-cell-overview.component.ts

```diff
@@ -127,4 +142,17 @@ export class TableCellOverviewComponent
       voucherReferenceId: referenceId,
     };
   });
+
+  getLabelByFieldName(fieldName: string | undefined): string | undefined {
```

Comment:
This needs to be computed somehow, otherwise (I think) it won't react to changes in `this.projectAttributes.isSuccess()`